lib/osdep/linux.c

```diff
@@ -2275,7 +2273,6 @@ static int do_linux_open(struct wif * wi, char * iface)
 	close(dev->fd_in);
 	if (acpi) fclose(acpi);
 	if (iface_malloced) free(iface);
-	if (iwpriv) free(iwpriv);
```

Comment:
this line should be deleted/commented as iwpriv points to a static char array

Reply:
Same as above.
This line should be deleted/comented, agreed.
The iwpriv will not exist anymore after this patch merges. All stuff return by wiToolsPath should be free, wiToolsPath uses malloc inside.